src/Storages/MergeTree/MergeTreeDataPartWide.cpp

```diff
@@ -185,9 +186,11 @@ void MergeTreeDataPartWide::checkConsistency(bool require_part_metadata) const
             {
                 getSerialization(name_type.name)->enumerateStreams([&](const ISerialization::SubstreamPath & substream_path)
                 {
-                    String file_name = ISerialization::getFileNameForStream(name_type, substream_path);
-                    String mrk_file_name = file_name + marks_file_extension;
-                    String bin_file_name = file_name + DATA_FILE_EXTENSION;
+                    String full_stream_name = ISerialization::getFileNameForStream(name_type, substream_path);
+                    String stream_name = checksums.getFileNameOrHash(full_stream_name);
```

Comment:
It's too easy to add a new usage of `getFileNameForStream` and forget about `getFileNameOrHash`. And tests will not catch it.

It's not clear what's the best way to avoid this (considering that `getFileNameForStream` is not only for MergeTree), but maybe we could introduce another function (that combines both) for MergeTree and restrict usages of `getFileNameForStream` with `check-style.sh`